python/ray/__init__.py

```diff
@@ -19,6 +20,10 @@
                  "override this by explicitly setting OMP_NUM_THREADS.")
     os.environ["OMP_NUM_THREADS"] = "1"
 
+if sys.platform == "darwin" and not gethostname().endswith(".local"):
+    logger.warning("The GCS Service may not correctly resolve your hostname."
```

Comment:
Many users will not know about the GCS - can we just say `The ray control service`?

Reply:
Also, don't forget to put a space at the end of the period

Reply:
Sounds good

Reply:
What do you mean by that wrt to the period.

Reply:
Try printing the logging statement as is; you’ll notice its not formatted correctly.

Reply:
Oh! yikes, totally forgot about that.